src/styles/header.scss

```diff
@@ -0,0 +1,113 @@
+header {
+    position: fixed;
+    z-index: 999;
+    width: 100%;
+    height: 3rem;
+    min-height: 50px;
+    color: #fff;
+    background: rgba(0, 8, 81, 0.8);
+    border: 1px solid transparent;
+    border-radius: 0;
+
+    .input-group {
+      float: right;
```

Comment:
Could you please replace `float `to another css properties? With `float ` the element is removed from the normal flow of the page